scala-package/pom.xml

```diff
@@ -393,6 +393,13 @@
       <groupId>commons-codec</groupId>
       <artifactId>commons-codec</artifactId>
       <version>1.10</version>
+      <scope>provided</scope>
```

Comment:
This should not be provided.

To exclude the dependencies to be packaged into jar file, you should add them in `exclude` section in assembly.xml:
```
<excludes>
...
        <exclude>commons-io:commons-io</exclude>
        <exclude>commons-codec:commons-codec</exclude>
<excludes>
```

Reply:
I originally had that but changed to provided scope. I was looking at the website check failure and I think it would be resolved by removing the provided scope and explicitly excluding the JARs. I'll take a look.

Reply:
Done